aclabs/todo/schema.py

```diff
@@ -68,8 +69,24 @@ class Query(object):
         name=graphene.String()
     )
 
+    def _get_text_filter(self, value):
+        return Q(text__istartswith=value) if value else None
```

Comment:
you can return Q() instead of None, and then in resolve_all_todos, you can apply & for _filters even if the there's only one filter sent